node/service/src/block_sealing.rs

```diff
@@ -240,8 +242,9 @@ pub async fn run_seal_command<B, BI, CB, E, C, TP, SC, CS, CIDP, P>(
 	while let Some(command) = commands_stream.next().await {
 		match command {
 			EngineCommand::SealNewBlock { create_empty, finalize, parent_hash, sender } =>
-				if sealing_create_empty_blocks ||
-					(!sealing_create_empty_blocks && pool.ready().count() > 0)
+				if sealing_mode != SealingMode::Interval ||
+					sealing_create_empty_blocks ||
+					pool.ready().count() > 0
```

Comment:
Just a comment to follow this logic:

- If we are in a mode *other* than instant sealing, form a block
- If we are in instant sealing AND we want to create empty blocks OR the count is > 0, form a block.

Reply:
Almost.  I think you meant interval sealing.